sdk/identity/azure-identity/test/ut/azure_cli_credential_test.cpp

```diff
@@ -60,6 +61,8 @@ class AzureCliTestCredential : public AzureCliCredential {
     return m_command;
   }
 
+  int GetLocalTimeToUtcDiffSeconds() const override { return m_localTimeToUtcDiffSeconds; }
```

Comment:
This isn't being used anywhere (and I am not sure why it's private).

Reply:
TokenCredentialImpl calls it: https://github.com/Azure/azure-sdk-for-cpp/pull/5179/files#diff-cc522e423ed15512dfbcccd5a21748c22c948f10b8b779f8201718aed16f98fcR353

This is private virtual, which means that the base class may still call it, but not the classes who inherit AzureCliTestCredential (should there be any).

Reply:
Oh right, it's virtual (hence the override). Got it.